client/my-sites/plugins/plugin-site-list/index.jsx

```diff
@@ -15,50 +13,52 @@ import isConnectedSecondaryNetworkSite from 'calypso/state/selectors/is-connecte
 
 import './style.scss';
 
-export class PluginSiteList extends Component {
-	static propTypes = {
-		plugin: PropTypes.object,
-		sites: PropTypes.array,
-		sitesWithSecondarySites: PropTypes.array,
-		title: PropTypes.string,
-	};
+export function PluginSiteList( props ) {
```

Comment:
we don't need to `export` here since we export with `export default PluginSiteList;` on the bottom.